pkg/client/clientcmd/loader_test.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clientcmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/v2/yaml"
+)
+
+var (
+	testConfigAlfa = Config{
```

Comment:
> Totally optional nit: s/alfa/alpha

Nato phonetics (alfa, bravo) as opposed to Greek letters (alpha, beta).  Juliett is spelled weird too, but I haven't gotten that far in recent tests: http://en.wikipedia.org/wiki/NATO_phonetic_alphabet.